### PR TITLE
Updated version of Feature/logout

### PR DESCRIPTION
**Pull request by Nazaqat**

### CODE REVIEW

File: src/client/components/logout/logout.component.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      type="submit"
+      onClick="location.href='/login?dis=yes'"
+    >
+      Logout
+    </button>
+    <br />
+    <button
+      type="submit"
+      className="confirm-cancel"
+      onClick={() => {
```
Comment (senner008) [inline src/client/components/logout/logout.component.js]: onClick={closeAction}

File: src/client/components/logout/logout.component.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    </>
+  );
+};
+
+const LogoutActions = ({ closeAction }) => (
+  <div className="actions">
+    <button
+      className="confirm-logout"
+      type="submit"
+      onClick="location.href='/login?dis=yes'"
```
Comment (senner008) [inline src/client/components/logout/logout.component.js]: onClick={() => {}}

File: src/client/components/logout/logout.component.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  const popupView = (closeAction) => (
+    <PopupModal title="LOG OUT" closeAction={closeAction}>
+      <LogoutContent>
+        <LogoutActions closeAction={closeAction} />
+      </LogoutContent>
+    </PopupModal>
+  );
+
+  return (
+    <Popup trigger={PopupTrigger} modal="true">
```
Comment (senner008) [inline src/client/components/logout/logout.component.js]: modal
(without the true)

Comment (Nazaqat) [reply src/client/components/logout/logout.component.js]: Means modal=""  or <Popup trigger={PopupTrigger} > ?

Comment (senner007) [reply src/client/components/logout/logout.component.js:18]: just modal

**Comment by Nazaqat** [comment]
ll be updated a new one

**Pull request closed without merging**